### PR TITLE
Build redistributable clients for alt-arch

### PR DESCRIPTION
**Pull request by detiber**
Build redistributable linux clients for more than just amd64

### CODE REVIEW

**Comment by detiber** [comment]
@smarterclayton This PR adds support for building redistributable binaries for supported linux targets other than amd64 (ppc64le, arm64, s390x).

This updates the building of the binaries and would allow for them to be published with the official github releases. I have a followup PR ready to also add the generated binaries to the RPM builds (at least for the CentOS PaaS SIG builds).

File: hack/build-cross.sh [hunk 1]
```diff
@@ -4,43 +4,32 @@
...
 
 # Set build tags for these binaries
 readonly OS_GOFLAGS_TAGS="include_gcs include_oss containers_image_openpgp"
 readonly OS_GOFLAGS_TAGS_$(os::build::platform_arch)="gssapi"
 
 # by default, build for these platforms
-platforms=(
-  linux/amd64
+platforms=( )
+for arch in "${OS_BUILD_SUPPORTED_ARCHES[@]}"; do
```
Comment (smarterclayton) [inline hack/build-cross.sh]: The should be server platforms and should not be linux specific.  We will eventually build on non-linux this way.

**Comment by detiber** [comment]
@smarterclayton updated to better break up server and client binaries, and also added the needed changes to be able to consume the changes within the spec file.

**Pull request closed without merging**